### PR TITLE
[update-dependencies] Copy repo into image vs using volume mount

### PR DESCRIPTION
**Pull request by MichaelSimons**
This is needed in order to reliably run the Maestro build hook.

### CODE REVIEW

**Comment by ravimeda** [comment]
@dotnet-bot test NanoServer-sac2016 - 1. Dockerfiles please

**Comment by ravimeda** [comment]
@dotnet-bot test NanoServer-1709 - 2.1 Dockerfiles please

File: update-dependencies/Invoke-UpdateDependencies.ps1 [hunk 1]
```diff
@@ -13,13 +13,13 @@ $ErrorActionPreference = 'Stop'
 $imageName = "update-dependencies"
 
 try {
-    & docker build -t $imageName -f $PSScriptRoot\Dockerfile --pull $PSScriptRoot
+    & docker build -t $imageName -f $PSScriptRoot\Dockerfile --pull $PSScriptRoot\..
```
Comment (ravimeda) [inline update-dependencies/Invoke-UpdateDependencies.ps1]: Minor - replace this `$PSScriptRoot\..` with `$repoRoot` assuming the declaration from line 21 is moved up,

Comment (MichaelSimons) [reply update-dependencies/Invoke-UpdateDependencies.ps1]: Good suggestion - fixed.

**Pull request merged**